lib/components/atoms/Blockquote/Blockquote.story.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { storiesOf } from '@storybook/react';
+// import { action } from '@storybook/addon-actions';
+import defaultConfig from './Blockquote.mock';
+
+// Import Styled Component to showcase variations
+import Blockquote, { BlockquoteVanilla } from '.';
+
+// TODO: indentation is not working fine in editor. Need to check for .hbs files
+
+storiesOf('Atoms/Blockquote', module)
+  .addParameters({
+    jest: ['Blockquote', 'BlockquoteVanilla'],
+  })
+  .add('Blockquote Knobs', () => (
+    <BlockquoteVanilla
+      {...defaultConfig}
+      className={`hide-default-sample ${defaultConfig.className}`}
```

Comment:
This class `hide-default-sample  is no longer needed

Reply:
fixed.